--- travis-ci-go-precise-1480652642/env
+++ travis-ci-go-precise-1485379378/env
@@ -3,9 +3,9 @@
 PACKER_BUILDER_TYPE=googlecompute
 PACKER_BUILD_NAME=googlecompute
 PACKER_TEMPLATES_BRANCH=master
-PACKER_TEMPLATES_SHA=10d938a
-TAGS=dist:precise,feature_basic:true,feature_chromium:true,feature_firefox:true,feature_google-chrome:true,feature_memcached:true,feature_mongodb:true,feature_mysql:true,feature_phantomjs:true,feature_postgresql:true,feature_rabbitmq:true,feature_redis:true,feature_sphinxsearch:true,feature_sqlite:true,feature_xserver:true,language_go:true,os:ubuntu,packer_chef_time:20161202T042525Z
+PACKER_TEMPLATES_SHA=294c7d9
+TAGS=dist:precise,feature_basic:true,feature_chromium:true,feature_firefox:true,feature_google-chrome:true,feature_memcached:true,feature_mongodb:true,feature_mysql:true,feature_phantomjs:true,feature_postgresql:true,feature_rabbitmq:true,feature_redis:true,feature_sphinxsearch:true,feature_sqlite:true,feature_xserver:true,language_go:true,os:ubuntu,packer_chef_time:20170125T212516Z
 TRAVIS_COOKBOOKS_BRANCH=
 TRAVIS_COOKBOOKS_DIR=/tmp/chef-stuff/travis-cookbooks
 TRAVIS_COOKBOOKS_EDGE_BRANCH=precise-stable
-TRAVIS_COOKBOOKS_SHA=9c6cd11
+TRAVIS_COOKBOOKS_SHA=479d954
