--- travis-ci-connie-trusty-1480957854/env
+++ travis-ci-connie-trusty-1485370628/env
@@ -3,8 +3,8 @@
 PACKER_BUILDER_TYPE=googlecompute
 PACKER_BUILD_NAME=googlecompute
 PACKER_TEMPLATES_BRANCH=master
-PACKER_TEMPLATES_SHA=0f241c1
-TAGS=dist:trusty,feature_basic:true,feature_disabled-ipv6:true,feature_docker:true,feature_docker-compose:true,feature_go-toolchain:true,feature_perl_interpreter:true,feature_perlbrew:true,feature_python_interpreter:true,feature_ruby_interpreter:true,language___connie__:true,language_bash:true,language_generic:true,language_minimal:true,language_sh:true,language_shell:true,os:ubuntu,packer_chef_time:20161205T171229Z
+PACKER_TEMPLATES_SHA=294c7d9
+TAGS=dist:trusty,feature_basic:true,feature_disabled-ipv6:true,feature_docker:true,feature_docker-compose:true,feature_go-toolchain:true,feature_perl_interpreter:true,feature_perlbrew:true,feature_python_interpreter:true,feature_ruby_interpreter:true,language___connie__:true,language_bash:true,language_generic:true,language_minimal:true,language_sh:true,language_shell:true,os:ubuntu,packer_chef_time:20170125T185829Z
 TRAVIS_COOKBOOKS_BRANCH=
 TRAVIS_COOKBOOKS_DIR=/tmp/chef-stuff/travis-cookbooks
-TRAVIS_COOKBOOKS_SHA=998099c
+TRAVIS_COOKBOOKS_SHA=9ac3e46
